tlatools/org.lamport.tlatools/src/tla2sany/xml/XMLExporter.java

```diff
@@ -41,13 +41,7 @@
 
 public class XMLExporter {
 
-  static final String JAXP_SCHEMA_LANGUAGE =
-        "http://java.sun.com/xml/jaxp/properties/schemaLanguage";
-  static final String W3C_XML_SCHEMA =
-        "http://www.w3.org/2001/XMLSchema";
-  static final String TLA_SCHEMA = "http://tla.msr-inria.inria.fr/tlaps/sany.xsd";
-  static final String JAXP_SCHEMA_SOURCE =
-        "http://java.sun.com/xml/jaxp/properties/schemaSource";
+  static final String W3C_XML_SCHEMA = "http://www.w3.org/2001/XMLSchema";
```

Comment:
Could we just package `XMLSchema` as part of `tla2tools.jar` and just do offline validation?

Reply:
We could... I am trying to figure out whether it actually fetches that URL or the library embeds a bunch of these well-known schemas to avoid that.

Reply:
I asked about this on stackoverflow so we will see: https://stackoverflow.com/questions/79145804/does-javas-schemafactory-actually-fetch-the-xml-schema-from-w3-org

Reply:
Person on stackoverflow answered saying it does not hit the network. I suppose we can believe them.